app.py

```diff
@@ -376,18 +377,12 @@ def generate_graph(street, direction, day_type='Weekday', period='AMPK'):
                    ],
                            className='four columns'
                           ),
-                  html.Div(children=[
-                    html.H2(id=STREETNAME_DIV[0], children=[html.B('Dundas Eastbound:'),
-                                                            ' Bathurst - Jarvis']),
-                    dcc.Graph(id=GRAPHS[0],
-                              config={'displayModeBar': False}),
-                    html.H2(id=STREETNAME_DIV[1], children=[html.B('Dundas Westbound:'),
-                                                            ' Jarvis - Bathurst']),
-                    dcc.Graph(id=GRAPHS[1],
-                              config={'displayModeBar': False})
-                ],
-                         className='eight columns'
-                        ),
+    html.H2(id=STREETNAME_DIV[0], children=[html.B('Dundas Eastbound:'),
+                                                ' Bathurst - Jarvis']),
+    html.Div(id = GRAPHS[0], className='eight columns'),
+    html.H2(id=STREETNAME_DIV[1], children=[html.B('Dundas Westbound:'),
+                                                ' Jarvis - Bathurst']),
+    html.Div(id = GRAPHS[1], className='eight columns')
```

Comment:
The id for the `html.Div` should be distinct from the id for the `dcc.Graph`

The Graph doesn't appear to be disappearing in this implementation.

![image](https://user-images.githubusercontent.com/10623872/33769295-97d3bbee-dbf7-11e7-8313-0880b0245464.png)